src/classes/Hotel.js

```diff
@@ -0,0 +1,48 @@
+import Room from './Room';
+import Booking from './Booking';
+import Customer from './Customer';
+import dayjs from 'dayjs';
+import isBetween from 'dayjs/plugin/isBetween';
+
+dayjs.extend(isBetween)
+
+class Hotel {
+  constructor(roomsData, bookingsData, customersData) {
+    this.rooms = roomsData;
+    this.bookings = bookingsData;
+    this.customers = customersData;
```

Comment:
If time permits, manager property would go here, and it would create a new instance of the Manager class.